hedera-node/src/test/java/com/hedera/services/state/merkle/MerkleUniqueTokenSerdeTest.java

```diff
@@ -21,26 +21,39 @@
  */
 
 import com.hedera.services.state.submerkle.EntityId;
+import com.hedera.services.utils.NftNumPair;
 import com.hedera.test.serde.SelfSerializableDataTest;
 import com.hedera.test.utils.SeededPropertySource;
 
+import static com.hedera.services.state.merkle.MerkleUniqueToken.RELEASE_0180_VERSION;
+import static com.hedera.services.state.merkle.MerkleUniqueToken.RELEASE_0250_VERSION;
+
 public class MerkleUniqueTokenSerdeTest extends SelfSerializableDataTest<MerkleUniqueToken> {
+	public static final int NUM_TEST_CASES = 2 * MIN_TEST_CASES_PER_VERSION;
+
 	@Override
 	protected Class<MerkleUniqueToken> getType() {
 		return MerkleUniqueToken.class;
 	}
 
 	@Override
 	protected MerkleUniqueToken getExpectedObject(final SeededPropertySource propertySource) {
-		return propertySource.nextMerkleUniqueToken();
+		return propertySource.next0260UniqueToken();
 	}
 
 	@Override
 	protected MerkleUniqueToken getExpectedObject(final int version, final int testCaseNo) {
-		var expected = super.getExpectedObject(version, testCaseNo);
-		if (version < MerkleUniqueToken.RELEASE_0250_VERSION) {
-			expected.setSpender(EntityId.MISSING_ENTITY_ID);
+		final var propertySource = SeededPropertySource.forSerdeTest(version, testCaseNo);
+		final var seededObject = getExpectedObject(propertySource);
+		if (version <= RELEASE_0180_VERSION) {
+			seededObject.setSpender(EntityId.MISSING_ENTITY_ID);
+		}
+		if (version <= RELEASE_0250_VERSION) {
```

Comment:
and this < RELEASE_0260_VERSION

Reply:
would be same right.

Reply:
Yeah just for readability, similar to the code. But this should be the same.